tutugodfrey/maintenance-tracker
# chore(test-signup): write test for signup endpoint

#### What does this PR do?
test signup endpoint

#### Description of Task to be completed?
/api/v1/users/signup is tested and working 

#### How should this be manually tested?
follow instruction in project README on how to run test
ensure to set .env variable as specified in .env.example file

### What are the relevant pivotal tracker stories?
#157781703
